extensions/wikia/ARecoveryEngine/js/PageFair/recovery.js

```diff
@@ -29,7 +29,7 @@ define('ext.wikia.aRecoveryEngine.pageFair.recovery', [
 	}
 
 	function isBlocking() {
-		var isBlocking = !!(win.ads && win.ads.runtime.pf && win.ads.runtime.pf.blocking);
+		var isBlocking = !!(win.ads && win.ads.runtime.pf && win.ads.runtime.pf.recovering);
```

Comment:
As far as I understand it's needed to run btf queue ([btfBlocker.js#L100](https://github.com/Wikia/app/blob/baa49db61952546ef3f034576481f4943b4ef604/extensions/wikia/AdEngine/js/provider/btfBlocker.js#L100)). Wouldn't it be safer to start btf queue (`btfBlocker::startBtfQueue`) in `adonis.od` callback?